homeassistant/components/iss/binary_sensor.py

```diff
@@ -107,6 +108,7 @@ def extra_state_attributes(self):
         if self.iss_data:
             attrs = {
                 ATTR_ISS_NUMBER_PEOPLE_SPACE: self.iss_data.number_of_people_in_space,
+                ATTR_ISS_PEOPLE_IN_SPACE: self.iss_data.people_in_space.get("people"),
```

Comment:
This will add quite a bit of data to our state machine and database history.

What would the use case be?

Reply:
Ah, I did not imagine all this, sorry. Curiosity. Simply curiosity. :(

Reply:
I think this should be a separat sensor. I was working on separating all the attributes out onto different sensors. But that stalled as i am moving in a couple of weeks.